src/components/TimeInput.vue

```diff
@@ -126,16 +126,14 @@ export default {
       const value = event.target.value
       this.hour = value ? value : null;
       this.writeModel();
-      this.$emit('changeHour', this.hour);
     },
-    changeMinuteHandler() {
+    changeMinuteHandler(event) {
       const value = event.target.value;
       this.minute = value ? value : null;
       this.writeModel();
```

Comment:
Hey, just had a question here. Is the 'changeHour' event no longer needed/unused? Or just not really necessary to emit on creation?

Reply:
It was unused, same as the changeMinute event